lib/napa_rabbit_publisher/amqp_singleton.rb

```diff
@@ -5,7 +5,7 @@ class AMQPSingleton
     attr_reader :connection, :channel, :exchange
     # we only want one connection at a time (most of the time) to RabbitMQ
     def initialize
-      @connection = Bunny.new(ENV['AMQP_HOST'])
+      @connection = Bunny.new(ENV['CLOUDAMQP_URL'] || ENV['AMQP_HOST'])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.